### PR TITLE
Connect dashboard exops and events apis to directory-api

### PR DESCRIPTION
**Pull request by WillTaylor22**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.
 - [x] (if adding env vars) Added any new environment variable to vault.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -2,9 +2,10 @@
...
 # I201 Missing newline between import groups
 # I202 Additional newline in a group of imports
+# N806 variable 'User' in function should be lowercase
 
 [flake8]
 exclude=.venv,venv,**migrations**,node_modules
 max-line-length=120
 max-complexity = 7
-ignore = I100, I101, I201, I202
+ignore = I100, I101, I201, I202, N806
```
Comment (sdonk) [inline .flake8]: We don't want to ignore this globally, just add a noqa at the offended line

File: config/env/dev [hunk 2]
```diff
@@ -25,7 +25,13 @@ SSO_OAUTH2_LINKEDIN_URL=http://sso.trade.great:8004/sso/accounts/login/via-linke
 SSO_OAUTH2_GOOGLE_URL=debug
 DIRECTORY_FORMS_API_BASE_URL=http://forms.trade.great:8011
 DIRECTORY_FORMS_DEFAULT_TIMEOUT=30
+<<<<<<< HEAD
```
Comment (sdonk) [inline config/env/dev]: merge debris?

File: core/helpers.py [hunk 3]
```diff
@@ -65,69 +66,22 @@ def create_user_profile(data, sso_session_id):
...
-            'url': '#',
-            'published_data': '03 January 2020',
-            'closing_data': '05 June 2020',
-        }
-    ]
+    search_term = ' '.join(sectors)
+    results = api_client.personalisation.export_opportunities_by_relevance_list(sso_session_id, search_term)
+    if (results.status_code == 200):
+        return parse_opportunities(results.json()['results'])
+    else:
```
Comment (sdonk) [inline core/helpers.py]: you can remove else and move the next return one indentation back

File: core/urls.py [hunk 4]
```diff
@@ -22,8 +22,9 @@ def anonymous_user_required(function):
         name='landing-page'
     ),
     path(
-        'dashboard/',
-        login_required(core.views.DashboardView.as_view(), login_url='/'),
+        'dashboard/', core.views.DashboardView.as_view(),
+        # login_required(
```
Comment (sdonk) [inline core/urls.py]: Does this need to be uncommented?

File: core/helpers.py (hunk 3 above)
Comment (WillTaylor22) [reply core/helpers.py]: Nice thanks

File: config/env/dev (hunk 2 above)
Comment (WillTaylor22) [reply config/env/dev]: Oops! Thanks

File: .flake8 (hunk 1 above)
Comment (WillTaylor22) [reply .flake8]: Ah got it. Thx

File: core/urls.py (hunk 4 above)
Comment (WillTaylor22) [reply core/urls.py]: Yup

**Pull request merged**